src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferencesSearchEngine.cs

```diff
@@ -24,7 +24,7 @@ internal partial class FindReferencesSearchEngine
         private readonly Solution _solution;
         private readonly IImmutableSet<Document> _documents;
         private readonly ImmutableArray<IReferenceFinder> _finders;
-        private readonly StreamingProgressTracker _progressTracker;
+        private readonly IStreamingProgressTracker _progressTracker;
         private readonly IStreamingFindReferencesProgress _progress;
```

Comment:
How does this field relate to this whole thing? Are there *multiple* progress reporting things going on here?

Reply:
Or we could just delete this field and replace all uses with _progress.ProgressTracker if we wanted?

Reply:
i'm fine with doing that.  it will likely be clearer.